src/Scripts/DockerLocalBuild.ps1

```diff
@@ -12,23 +12,37 @@
 #       "musl" CLib for the Alpine 3.7 sample. More generally, use "gnu" build output for glibc-
 #       based distros and "musl" build output for musl-libc-based distros.
 
+<#
+.SYNOPSIS
+ Launch a local Linux build using docker.
+#>
 param
 (
+    # Optionally change the enlistment root. 
     [Parameter(Mandatory=$false)]
     [ValidateNotNullOrEmpty()]
     [ValidateScript({$_ -and (Test-Path $_)})]
     [String] $EnlistmentRoot = $(Resolve-Path -Path "$PSScriptRoot\..\.."),
 
+    # Indicates what version of the c runtime will be used for the build.
+    # This determines the version of Linux that will be used for the build.
     [Parameter(Mandatory=$false)]
     [ValidateNotNullOrEmpty()]
     [ValidateSet('gnu', 'musl')]
     [String] $CLib = 'gnu',
 
+    # What flavor to build. Debug or release. Default is Debug.
     [Parameter()]
     [ValidateNotNullOrEmpty()]
     [ValidateSet('Debug', 'Release')]
     [String] $Type="Debug",
 
+    # Use a local docker image. Default is to download the official image
+    # that is used for released builds.
+    [Parameter()]
+    [Switch] $LocalDockerImage,
```

Comment:
nit: BuildDockerImage, the name LocalDockerImage almost sounds like I should point it to a local Docker image.

Reply:
I thought about that, but I didn't want to use BuildDockerImage since it only builds the local image if it doesn't already exist.

Reply:
BuildLocalImage? BuildImageLocally?

Reply:
That command didn't seem to work. But if I pipe the content of the file to docker build, then I don't need to set a context. I'll do that instead.